### PR TITLE
Two New Zam Products v2

### PR DESCRIPTION
**Pull request by Optimism333**
<content>

## The Return of Zam ##

This is a quick Zam update mostly aimed at introducing two new products, with two small tweaks included.
* Minor Tweak: Reordered the name of Radical Zam Biscuit. It's now named Zam Radical Biscuit
* Minor Tweak: Removed the sweet flag from the Zam Radical Biscuit, since it has no sweet ingredients.

## New Products ##
Screenshot (724)
 * **Zam Buttery Biscuit:** A new premium biscuit synthesized with butter coating. Administrator Zam's favorite! (Price = A coin) (Reagents = Nutriment, Hyronalin, Butter, Sugar, Sulphuric Acid) (Number Stocked = 2)
 * **Zam Human Hydrator:** Zam labs have replaced the generic mineral water previously stocking the machine with a new and improved version. It contains five essential minerals for the ultimate human hydration! (Price = 40 credits) (Reagents = Water, Iron, Copper, Silver, Gold, Diamond) (Number Stocked = 6)

:cl:
 * rscadd: New premium Zam product, a biscuit with butter!
 * rscadd: New Zam beverage, Human Hydrator! A premium mineral water for nongreys.

### CODE REVIEW

**Comment by Optimism333** [comment]
## Notes on v2 ##

* Zam Buttery Biscuit is now identical to the normal zam biscuit, except more nutritious since the cocoa is replaced with liquid butter.
* After considering feedback on the pr, I decided to leave the Human Hydrator alone, since most seemed to agree it was fine as is.

File: code/modules/reagents/reagent_containers/food/drinks.dm [hunk 1]
```diff
@@ -1108,6 +1108,21 @@
...
+	icon_state = "Zam_HumanHydrator"
+/obj/item/weapon/reagent_containers/food/drinks/soda_cans/zam_humanhydrator/New()
+	..()
+	reagents.add_reagent(WATER, 35)
+	reagents.add_reagent(IRON, 1)
+	reagents.add_reagent(COPPER, 1)
+	reagents.add_reagent(SILVER, 1)
+	reagents.add_reagent(GOLD, 1)
+	reagents.add_reagent(DIAMONDDUST, 1)
+	src.pixel_x = rand(-10, 10) * PIXEL_MULTIPLIER
```
Comment (Kurfursten) [inline code/modules/reagents/reagent_containers/food/drinks.dm]: no src

Comment (Optimism333) [reply code/modules/reagents/reagent_containers/food/drinks.dm]: What do you mean? I'm not sure what I'm supposed to correct. Do I remove the src lines?

Comment (Kurfursten) [reply code/modules/reagents/reagent_containers/food/drinks.dm]: delete the `src.` in front of pixel_x and pixel_y

Comment (Optimism333) [reply code/modules/reagents/reagent_containers/food/drinks.dm]: @Kurfursten Alright, I did it.

**Pull request merged**